vsock-bridge/src/channel.cpp

```diff
@@ -0,0 +1,13 @@
+#include <channel.h>
+
+namespace vsockio
+{
+    void DirectChannel::performIO()
+    {
+        _a->readInput();
```

Comment:
Trying all 4 operations when a channel is ready does add more system calls. I like this tradeoff for simplicity.

Reply:
Yep, added a comment to make this a bit clearer.